packages/dso-toolkit/src/components/form/form-groups/files/form-group-files.scss

```diff
@@ -11,6 +11,14 @@
 @use "../../form.mixins" as css-form-mixins;
 
 .form-group.dso-files {
+  // if file list is empty
+  &:not(:has(.dso-filelist li)) {
```

Comment:
We hebben het er in de chat al even over gehad, maar ik denk dat we dit niet zo moeten doen.

De verantwoordelijkheid voor het renderen van correcte markup ligt bij de afnemer. Als er geen lijst moet worden getoond omdat er geen items zijn, dan moet de afnemer geen lijst renderen.

Je gaf aan "extra laagje veiligheid", maar hebben wat is onze rol in dit geheel dan? Gaan wij voortaan "extra veiligheid" inbouwen? Ik vind dat geen goed idee omdat zowel afnemer als toolkit dan van dezelfde markup "wat gaat vinden". En daarnaast, wat als afnemers gaan leunen op deze extra veiligheid waardoor het "verwacht gedrag" wordt?

Ik stel voor dat we dit met documentatie en markup voorschriften gaan oplossen.

* `packages/dso-toolkit/src/components/form/readme.md` uitbreiden met een extra kop voor Form Group Files
* `storybook/src/components/form/form-group-files.css-template.ts:36` uitbreiden met een voorwaarde op `formGroup.files.length > 0`.

cc: @iterox

Reply:
Het is qua scripting een bak extra werk als er gecheckt moet worden of er al een `ul` in de DOM aanwezig is (zo nee, maak er eentje en zet het itejm erin, zo ja, zet het item erin. Bij weghalen: was dit de laatste? zo ja, haal lijst weg. als men die `ul` erin kan laten staan als placeholder (wat in mijn ervaring best vaak voorkomt) dan scheelt dat een hoop code. 
Mar goed, dat gezegd hebbende kan ik me er ook wel in vinden om wat minder mee te denken en dit aan de devs over te laten. de CSS is weg, dus daar is alles eigenlijk weer bij het oude ;)

Reply:
Dat wordt door de frameworks gerealiseerd. Wie gaat de templating aanpassen?

Reply:
I'm on it

Reply:
Gedaan

Reply:
AWESOME!